### PR TITLE
DIP-6: Proposing ticket system with prior implementations

### PR DESCRIPTION
**Pull request by colourful-land**
Forum Discussion is here: https://forum.devcon.org/t/attestation-based-ticketing-system-that-is-managed-by-ethereum-smart-contracts-and-integrated-with-pretix-re-rfp-1-onchain-ticketing/54

### CODE REVIEW

File: DIPs/DIP-6-Ticket.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+Any services which require to verify attendees, like a discount for products, redeemable vouchers, content unlocking, IoT(open door, control screen), collateral for DeFis and etc.
+
+The [CO2ken project](https://forum.devcon.org/t/co2ken-carbon-neutral-devcon/27) is an ideal team to collaborate with and to integrate their proposal (DIP-1) to provide attendees the choice of offesting some of the carbon emissions caused by the event. This functionality can be added to the DEVCON DAPP website and any users using the website can support the cause by donating tokens to CO2ken
+
+## Links & Additional Information
+
+**Possible Questions**
+1. Can the Tickets work without internet?
+A. Yes, the ticket can be used to check-in with out internet. This was one of the key requirements for the implemetnations for FIFA and UEFA. However, internet will be needed to access the advanced DAPP features offered by the ticket such as voting.
```
Comment (ligi) [inline DIPs/DIP-6-Ticket.md:117]:
```suggestion
A. Yes, the ticket can be used to check-in with out internet. This was one of the key requirements for the implementations for FIFA and UEFA. However, internet will be needed to access the advanced DAPP features offered by the ticket such as voting.
```

File: DIPs/DIP-6-Ticket.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+Furthermore, a ticket ownership proof can be generated from [TokenScript](http://docs.tokenscript.org/) that allows a website to recognise DEVCON ticket holders. This is useful for ticket holders to book services or to apply a discount. We can create a sample website for service providers to copy.
+
+**Previous implementations**
+
+Our first ticket implementation was on 2018, as an experiment with FIFA officials to tokenise FIFA World Cup 2018 VIP tickets. The experiment was successful, where 50 users received attestation links through email and 28 of them converted those attestations into Ethereum tokens.
+
+The next full fledged implmentation was in 2019, with Shankai Sports for tokenising EURO Championship 2020 VIP event tickets, with integrated hospitality passes([App](https://apps.apple.com/us/app/shankai/id1492559481). The tickets were tokenised and the apps were ready to go, but the event got cancelled 😭😭😭)
+
+In 2019, we also worked tordwards implementing EDCON 2020 event tickets on blockchain. Again, the app and tickets were ready, but thanks to the pandemic, the event got cancelled 😭😭😭)
```
Comment (ligi) [inline DIPs/DIP-6-Ticket.md:84]:
```suggestion
In 2019, we also worked towards implementing EDCON 2020 event tickets on blockchain. Again, the app and tickets were ready, but thanks to the pandemic, the event got cancelled 😭😭😭)   
```

File: DIPs/DIP-6-Ticket.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+Furthermore, a ticket ownership proof can be generated from [TokenScript](http://docs.tokenscript.org/) that allows a website to recognise DEVCON ticket holders. This is useful for ticket holders to book services or to apply a discount. We can create a sample website for service providers to copy.
+
+**Previous implementations**
+
+Our first ticket implementation was on 2018, as an experiment with FIFA officials to tokenise FIFA World Cup 2018 VIP tickets. The experiment was successful, where 50 users received attestation links through email and 28 of them converted those attestations into Ethereum tokens.
+
+The next full fledged implmentation was in 2019, with Shankai Sports for tokenising EURO Championship 2020 VIP event tickets, with integrated hospitality passes([App](https://apps.apple.com/us/app/shankai/id1492559481). The tickets were tokenised and the apps were ready to go, but the event got cancelled 😭😭😭)
```
Comment (ligi) [inline DIPs/DIP-6-Ticket.md:82]:
```suggestion
The next full fledged implementation was in 2019, with Shankai Sports for tokenising EURO Championship 2020 VIP event tickets, with integrated hospitality passes([App](https://apps.apple.com/us/app/shankai/id1492559481). The tickets were tokenised and the apps were ready to go, but the event got cancelled 😭😭😭)
```

**Review by ligi — CHANGES_REQUESTED** [review]
Thanks for the PR! Found some typos and please rename to DIP-6.md - maybe also consolidate the usage of Ethereum (currently lower case ethereum and upper case Ethereum are present)

**Comment by skylarweaver** [comment]
Also, one more comment could you please change "DEVCON" to "Devcon" ? 

Better to differentiate from DEF CON ;-)

**Comment by colourful-land** [comment]
> Thanks for the PR! Found some typos and please rename to DIP-6.md - maybe also consolidate the usage of Ethereum (currently lower case ethereum and upper case Ethereum are present)

Thanks & mentioned issues are addressed a few weeks ago but forgot to update PR.

**Comment by ligi** [comment]
sorry for the delayed merge - we are still fleshing out the rough edges of this brand new process. Good thing is that we still have some time to DevCon ;-)
We now have a checklist and I will go through it - once all is checked we will merge.

 * [ ] Correct markdown (e.g. headers with ##, header surrounded by ---)
 * [ ] syntax check
 * [ ] filename is DIP-X.md (with X being the correct DIP number)
 * [ ] header contains (in this order)
   * [ ] DIP
   * [ ] Title
   * [ ] Status
   * [ ] Theme(s)
   * [ ] Tag(s)
   * [ ] Author(s)
   * [ ] Resources Required (optional)
   * [ ] Discussion-to (with link to devcon forum)
   * [ ] Created (in the format year-month-day e.g 2020-4-20)
 * [ ] contains these sections:
   * [ ] Abtract
   * [ ] Rationale
   * [ ] Implementation
   * [ ] Operational Reuirements & Ownership
 * [ ] PR title prefixed with DIP-X

**Comment by colourful-land** [comment]
Requested changes made. Thanks

**Review by ligi — APPROVED** [review]

**Pull request merged**